scripts/parse_pricing_api.py

```diff
@@ -0,0 +1,303 @@
+"""Script that parses pricing data from Google's API.
+
+Intended to be run periodically for the terra-ui's estimated price UI. The
+output is something you could copy and paste into a javascript file.
+See https://cloud.google.com/billing/v1/how-tos/catalog-api for more detail
+on Google Cloud pricing information.
+
+Usage:
+1) Follow the instructions at the above URL to create an API key
+2) Run `python3 parse_pricing_api.py ${API_KEY}`
+3) Copy and paste the output to the appropriate Javascript file.
+"""
+import sys
+import requests
+
+# See https://cloud.google.com/billing/v1/how-tos/catalog-api on
+# using API keys
+if len(sys.argv) < 2:
+  print('Usage: `python3 parse_pricing_api.py ${API_KEY}`')
+  sys.exit()
+API_KEY = sys.argv[1]
+
+# See https://cloud.google.com/skus/sku-groups/on-demand-vms,
+# https://cloud.google.com/skus/sku-groups/on-demand-persistent-disk-hdd
+# on SKU groups
+GCE_SERVICE = '6F81-5844-456A'
+
+N1_STANDARD_GROUP = 'N1Standard'
+PD_STANDARD_GROUP = 'PDStandard'
+GPU_STANDARD_GROUP = 'GPU'
+
+NVIDIA_TESTLA_T4 = 'Nvidia Tesla T4'
```

Comment:
Are these supposed to be `*TESLA*` instead of `*TESTLA*`?

Reply:
Yes. Thank you. Fixed.